picard/ui/options/dialog.py

```diff
@@ -93,6 +95,44 @@
 from picard.ui.util import StandardButton
 
 
+class ErrorOptionsPage(OptionsPage):
+
+    def __init__(self, parent=None, errmsg='', from_cls=None):
+        # copy properties from failing page
+        self.NAME = from_cls.NAME
+        self.TITLE = from_cls.TITLE
+        self.PARENT = from_cls.PARENT
+        self.SORT_ORDER = from_cls.SORT_ORDER
+        self.ACTIVE = from_cls.ACTIVE
+        self.HELP_URL = from_cls.HELP_URL
+
+        super().__init__(parent)
+
+        msg = _(
+            "Error while loading option page '%s':\n"
+            "\n"
+            "%s"
+        ) % (_(from_cls.TITLE), errmsg)
+        self.error = _("This page failed to load")
+
+        layout = QtWidgets.QVBoxLayout(self)
+        widget = QtWidgets.QLabel()
+        widget.setTextFormat(QtCore.Qt.TextFormat.PlainText)
+        widget.setText(msg)
+        widget.setWordWrap(True)
+        layout.addWidget(widget)
+
+        widget = QtWidgets.QLabel(
+            _('Please report this issue on the <a href="%s">Picard bug tracker</a>.')
+            % PICARD_URLS['bug_tracker']
+        )
```

Comment:
I know that I initially suggested this link, but now I'm wondering if it might be more appropriate to direct the user to the "Troubleshooting" section of the documentation (`webbrowser2.open('troubleshooting')`) or the "Reporting a Bug" section of that page at https://picard-docs.musicbrainz.org/troubleshooting/troubleshooting.html#reporting-a-bug instead, because that would provide instructions about including logs and such.  I'm okay with whatever you decide, but I thought I would mention the alternative for consideration.

Reply:
Done in https://github.com/metabrainz/picard/pull/2436/commits/4346b771528ac5f5a243f3998a4206358faa1e73